packages/Select/src/Select.js

```diff
@@ -1,94 +1,136 @@
 import React from "react";
-import classNames from "classnames";
 import PropTypes from "prop-types";
+import { callAll, extractChildrenProps } from "@paprika/helpers";
 import * as constants from "@paprika/constants/lib/Constants";
+import SelectPropsCollector from "./SelectPropsCollector";
 import * as sc from "./Select.styles";
 
 const Select = React.forwardRef((props, ref) => {
-  const renderPlaceholder = () => {
-    const { placeholder } = props;
-    if (!placeholder) return null;
-    return (
-      <option disabled value="" key="placeholder">
-        {placeholder}
-      </option>
-    );
-  };
-
   const {
     a11yText,
-    className,
     children,
+    defaultValue,
     hasError,
     isDisabled,
     isReadOnly,
-    size,
+    onChange,
     placeholder,
+    size,
     value,
     ...moreProps
   } = props;
 
-  if (a11yText || placeholder) {
-    moreProps["aria-label"] = a11yText || placeholder;
+  const isControlled = value !== undefined;
+  const containerProps = extractChildrenProps(children, SelectPropsCollector);
+  const [hasNoValue, setHasNoValue] = React.useState(isControlled ? !value : !defaultValue);
```

Comment:
I don't really understand the `hasNoValue` state, and also why is in a negation format hasValue would be easier to negate than `!(hasNoValue)` you have on line 40.

Reply:
Oh, you're probably right that a negative boolean name is not a good practice.  This is all for the styling of the placeholder:
https://github.com/acl-services/paprika/blob/fcf0b79156f1526bd53951cfe7d0e2c349672d92/packages/Select/src/Select.js#L30

It needs to apply different styling when the placeholder item is selected.  While _writing_ it "hasNoValue" was easier to understand, but it may be harder to _read_ so I can change it.

Reply:
🙏